compaction/time_window_compaction_strategy.hh

```diff
@@ -55,7 +55,16 @@ public:
     time_window_compaction_strategy_options(const std::map<sstring, sstring>& options);
 
     static void validate(const std::map<sstring, sstring>& options, std::map<sstring, sstring>& unchecked_options);
-
+private:
+    static std::chrono::seconds validate_compaction_window_unit(const std::map<sstring, sstring>& options);
```

Comment:
Nitpick: I don't think you needed to add all these dozen validate_* function to the header files. Couldn't they just have been static functions in the .cc file?